lua/neoclip/telescope.lua

```diff
@@ -163,25 +182,31 @@ local function spec_from_entry(entry)
     return spec
 end
 
-local function make_display(entry)
-    local to_display = {entry.contents[1]}
+local function make_display(entry, typ)
+    local to_display = { macro_to_keycodes_if_needed(entry.contents, typ)[1] }
     if settings.content_spec_column then
         table.insert(to_display, {spec_from_entry(entry), "Comment"})
     end
     return displayer(to_display)
 end
 
-local function entry_maker(entry)
-    return {
-        display = make_display,
-        contents = entry.contents,
-        regtype = entry.regtype,
-        filetype = entry.filetype,
-        ordinal = table.concat(entry.contents, '\n'),
-        -- TODO seem to be needed
-        name = 'name',
-        value = 'value', -- TODO what to put value to, affects sorting?
-    }
+local function entry_maker(typ)
```

Comment:
```suggestion
local function make_entry_maker(typ)
```
just to clarify that it makes the `entry_maker` and is not the `entry_maker` itself :)

Reply:
Yeah, that makes sense!